store/dynamodb/dynamodb.go

```diff
@@ -259,7 +259,7 @@ func (ddb *DynamoDB) List(directory string, options *store.ReadOptions) ([]*stor
 	}
 
 	kvArray := []*store.KVPair{}
-	val := new(store.KVPair)
+	var val *store.KVPair
```

Comment:
I'm curious: is that something highlighted by the linter? I generally privilege the first syntax than using the var keyword. `var val` is a bit weird imo.

Reply:
in this case, you don't need to initiate a  `store.KVPair` because the value this completely ignored. `val` is created after.